enterprise_data/tests/test_utils.py

```diff
@@ -104,3 +104,94 @@ def get_dummy_enterprise_api_data(**kwargs):
+                'course_start': '2016-09-01T00:00:00Z',
+                'enterprise_user': 111,
+                'user_country_code': 'US',
+                'course_title': 'All about acceptance testing!',
+                'course_duration_weeks': '8',
+                'course_pacing_type': 'instructor_paced',
+                'user_username': 'test_user',
+                'enterprise_sso_uid': 'harry',
+                'enterprise_site_id': None,
+                'enterprise_id': kwargs.get('enterprise_id'),
+                'course_end': '2016-12-01T00:00:00Z',
+                'lms_user_id': 11,
+                'enterprise_name': 'Enterprise 1',
+                'letter_grade': 'Pass',
+                'user_account_creation_timestamp': '2015-02-12T23:14:35Z',
+                'passed_timestamp': '2017-05-09T16:27:34.690065Z',
+                'course_max_effort': 4,
+                'consent_granted': True,
+                'user_email': 'test@example.com',
+                'course_key': 'edX/Open_DemoX',
+                'coupon_name': 'Enterprise Entitlement Coupon',
+                'coupon_code': 'PIPNJSUK33P7PTZH',
+                'offer': 'Percentage, 100 (#1234)',
+                'current_grade': 0.80,
+                'course_price': '200.00',
+                'discount_price': '120.00',
+                'course_api_url': ('/enterprise/v1/enterprise-catalogs/ee5e6b3a-069a-4947-bb8d-d2dbc323396c'
+                                   '/courses/edX/Open_DemoX/edx_demo_course'),
+                'unenrollment_end_within_date': True,
+            }, {
+                'enrollment_created_timestamp': '2014-06-27T16:02:38Z',
+                'unenrollment_timestamp': '2016-09-05T16:02:38Z',
+                'user_current_enrollment_mode': 'verified',
+                'last_activity_date': '2017-06-23',
+                'progress_status': 'Failed',
+                'course_id': 'edX/Open_DemoX/edx_demo_course',
+                'id': 4,
+                'course_min_effort': 2,
+                'course_start': '2016-09-01T00:00:00Z',
+                'enterprise_user': 333,
+                'user_country_code': 'US',
+                'course_title': 'All about acceptance testing!',
+                'course_duration_weeks': '8',
+                'course_pacing_type': 'instructor_paced',
+                'user_username': 'test_user',
+                'enterprise_sso_uid': 'harry',
+                'enterprise_site_id': None,
+                'enterprise_id': kwargs.get('enterprise_id'),
+                'course_end': '2016-12-01T00:00:00Z',
+                'lms_user_id': 11,
+                'enterprise_name': 'Enterprise 1',
+                'letter_grade': None,
+                'user_account_creation_timestamp': '2015-02-12T23:14:35Z',
+                'passed_timestamp': None,
+                'course_max_effort': 4,
+                'consent_granted': True,
+                'user_email': 'test@example.com',
```

Comment:
both test user should not have the same `user_email` to better test email searching